python/pyproject.toml

```diff
@@ -21,4 +21,9 @@ requires = [
     "scikit-build>=0.13.1",
     "cmake>=3.23.1",
     "ninja",
+    "treelite==3.0.0",
+    "treelite_runtime==3.0.0",
```

Comment:
```suggestion
    "treelite==3.0.1",
    "treelite_runtime==3.0.1",
```

Reply:
We should use 3.0.1? isn't this different from other places?

Reply:
https://github.com/rapidsai/cuml/pull/5018

Reply:
@dantegd are we expecting a treelite 3.0.1 release? [It's not available yet on PyPI](https://pypi.org/project/treelite/) and until it is we can't use it as a dependency for wheels

Reply:
Looks like everything is working to me now.